compiler/testData/codegen/bytecodeText/forLoop/forInReversed/forInReversedRangeLiteral.kt

```diff
@@ -24,12 +23,12 @@ fun box(): String {
 }
 
 // 0 reversed
+// 0 iterator
 // 0 getStart
 // 0 getEnd
 // 0 getFirst
 // 0 getLast
 // 0 getStep
-// 0 IFEQ
-// 0 IF_ICMPEQ
-// 2 IF_ICMPLT
-// 1 LCMP
+// 2 IF_ICMP
```

Comment:
// 2 IF_ICMP[LG]T
// 1 IF[LG]T
// 3 IF

Reply:
Good suggestion, didn't consider that!